.github/workflows/preview.yml

```diff
@@ -0,0 +1,29 @@
+name: Deploy PR previews for demo apps
+run-name: ${{ github.actor }} is deploying a PR
+concurrency: preview-${{ github.ref }}
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: ['*demo*']
```

Comment:
Same here

Reply:
This gives us a preview of the demo apps on any branch named `*demo*` so we could leave as is.

Reply:
https://github.com/HubSpot/calling-extensions-sdk/pull/120